dev-tools/scripts/smokeTestRelease.py

```diff
@@ -537,106 +502,100 @@ def getDirEntries(urlString):
     return l
   else:
     links = getHREFs(urlString)
-    for i, (text, subURL) in enumerate(links):
+    for i, (text, subURL) in enumerate(links): # pylint: disable=unused-variable
       if text == 'Parent Directory' or text == '..':
         return links[(i+1):]
+  return None
+
 
-def unpackAndVerify(java, project, tmpDir, artifact, gitRevision, version, testArgs, baseURL):
+def unpackAndVerify(java, tmpDir, artifact, gitRevision, version, testArgs):
   destDir = '%s/unpack' % tmpDir
   if os.path.exists(destDir):
     shutil.rmtree(destDir)
   os.makedirs(destDir)
   os.chdir(destDir)
   print('  unpack %s...' % artifact)
-  unpackLogFile = '%s/%s-unpack-%s.log' % (tmpDir, project, artifact)
+  unpackLogFile = '%s/solr-unpack-%s.log' % (tmpDir, artifact)
   if artifact.endswith('.tar.gz') or artifact.endswith('.tgz'):
     run('tar xzf %s/%s' % (tmpDir, artifact), unpackLogFile)
   elif artifact.endswith('.zip'):
     run('unzip %s/%s' % (tmpDir, artifact), unpackLogFile)
 
   # make sure it unpacks to proper subdir
   l = os.listdir(destDir)
-  expected = '%s-%s' % (project, version)
+  expected = 'solr-%s' % version
   if l != [expected]:
     raise RuntimeError('unpack produced entries %s; expected only %s' % (l, expected))
 
   unpackPath = '%s/%s' % (destDir, expected)
-  verifyUnpacked(java, project, artifact, unpackPath, gitRevision, version, testArgs, tmpDir, baseURL)
+  verifyUnpacked(java, artifact, unpackPath, gitRevision, version, testArgs)
   return unpackPath
 
-LUCENE_NOTICE = None
-LUCENE_LICENSE = None
 SOLR_NOTICE = None
 SOLR_LICENSE = None
 
-def verifyUnpacked(java, project, artifact, unpackPath, gitRevision, version, testArgs, tmpDir, baseURL):
-  global LUCENE_NOTICE
-  global LUCENE_LICENSE
+def is_in_list(in_folder, files, indent=4):
+  for fileName in files:
+    print("%sChecking %s" % (" "*indent, fileName))
+    found = False
+    for f in [fileName, fileName + '.txt', fileName + '.md']:
+      if f in in_folder:
+        in_folder.remove(f)
+        found = True
+    if not found:
+      raise RuntimeError('file "%s" is missing' % fileName)
```

Comment:
`file_name` rather than `fileName` is preferred in python as far as i know and since this function is new maybe nice to follow the python convention for it, noting that the `in_folder` argument already does.

```suggestion
  for file_name in files:
    print("%sChecking %s" % (" "*indent, file_name))
    found = False
    for f in [file_name, file_name + '.txt', file_name + '.md']:
      if f in in_folder:
        in_folder.remove(f)
        found = True
    if not found:
      raise RuntimeError('file "%s" is missing' % file_name)
```

Reply:
Just a question - how do you add a code suggestion to a PR, so that the "commit suggestion" button appears? It's a really nice workflow...

Reply:
There's a lot of Java smell in our python scripts. Luckily my IDEA suggests some PEP standard changes now and then, so I try to add those two LF between functions and use underscore. But it could use a bigger cleanup for sure.